commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -129,6 +111,14 @@ protected void validateArgs() {
     }
   }
 
+  private void validatePositiveValue(final long value, final String fieldName) {
+    if (value < 1) {
```

Comment:
`value <= 1`

Reply:
If I change it to that then this won't allow 1 as a value. I only want to exclude zero and less than zero values as these don't make sense for these options.